testlist/sim00.dat

```diff
@@ -1,27 +1,21 @@
-/sim
+/sim/*/*/*/[a-n]*
+
+# macOS doesn't have ALSA
+-Darwin,sim:alsa
+
 -Darwin,sim:cxxtest
 -Darwin,sim:libcxxtest
 
-# CONFIG_SIM_M32=y
-# The recent versions of macOS is 64-bit only
+# Do not build Linux configs
+-Darwin,sim:linuxi2c
+
+# macOS doesn't support 32bit(CONFIG_SIM_M32=y) anymore
 -Darwin,sim:loadable
 -Darwin,sim:module32
--Darwin,sim:rpproxy
--Darwin,sim:rpserver
--Darwin,sim:sotest32
 
-# X11
 # macOS doesn't have X11
+-Darwin,sim:lvgl
 -Darwin,sim:nsh2
 -Darwin,sim:nx11
 -Darwin,sim:nxlines
 -Darwin,sim:nxwm
--Darwin,sim:touchscreen
--Darwin,sim:lvgl
-
-# macOS doesn't have ALSA
--Darwin,sim:alsa
```

Comment:
this was missed in the split version, which is causing the failure.

Reply:
actually i see it is there
maybe the syntax you're using to include configs is not supported?

Reply:
testbuild.sh fail to remove the dir from line:
```
dotest $i${line/$dir/}
``
https://github.com/apache/incubator-nuttx/pull/3298 fix it by adding "

Reply:
but don't you need the glob so that you can add the configs using `*`?

Reply:
config finding need glob:
```
list=`find boards$dir -name defconfig | cut -d'/' -f4,6`
```
but prefix removing don't need:
```
dotest $i${line/$dir/}
```

Reply:
got it